sktime/forecasting/base/_fh.py

```diff
@@ -536,7 +536,9 @@ def _to_relative(fh: ForecastingHorizon, cutoff=None) -> ForecastingHorizon:
 # with object methods, see B019 error of flake8-bugbear for a detail explanation.
 # See more here: https://github.com/alan-turing-institute/sktime/issues/2338
 @lru_cache(typed=True)
-def _to_absolute(fh: ForecastingHorizon, cutoff) -> ForecastingHorizon:
+def _to_absolute(
+    fh: ForecastingHorizon, cutoff, consecutive=False
+) -> ForecastingHorizon:
     """Return absolute version of forecasting horizon values.
 
     Parameters
```

Comment:
please, add `consecutive` to `Parameters` section of the docstring

Reply:
removed consecutive